uuid.go

```diff
@@ -82,11 +82,14 @@ const (
 )
 
 // Timestamp is the count of 100-nanosecond intervals since 00:00:00.00,
-// 15 October 1582 within a V1 UUID. This type has no meaning for other
-// UUID versions since they don't have an embedded timestamp.
+// 15 October 1582 within a V1 or V6 UUID, or as a common intermediate
+// representation of the (Unix Millisecond) timestamp within a V7 UUID.
+// This type has no meaning for other UUID versions since they don't
+// have an embedded timestamp.
 type Timestamp uint64
 
 const _100nsPerSecond = 10000000
+const _100nsPerMillisecond = 10000
```

Comment:
One more nit: not really a fan of defining a named constant that's only used in a single place as an implementation detail.  it breaks code locality.

Reply:
I thought about that, but I'd rather have both time conversion constants next to each other so future devs can see the relationship between them.

Reply:
counter-point: the name `_100nsPerMillisecond` doesn't convey what the value represents (there aren't 100 ns in a millisecond 🤨) or how it's used.  a casual reader would see this and have to grep the code to figure out why this constant exists.

fwiw I have the same gripe with `_100nsPerSecond` but that one precedes my being a maintainer.

Reply:
I it's not worth spending more time discussing this. Since there's already a comment that explains the conversion, I changed it to just use a "magic number" there instead of a constant.

Reply:
> I understand the names to perfectly convey what the value represents 

in a way this reinforces my gripe.  we both read the same code and interpreted the name differently. 😉